src/value.h

```diff
@@ -388,13 +386,20 @@ static inline uint32_t RSValue_ArrayLen(const RSValue *arr) {
   return arr ? arr->arrval.len : 0;
 }
 
-typedef enum {
-  SENDREPLY_FLAG_TYPED = 0x01,
-  SENDREPLY_FLAG_EXPAND = 0x02,
-} SendReplyFlags;
-
-/* Based on the value type, serialize the value into redis client response */
-int RSValue_SendReply(RedisModule_Reply *reply, const RSValue *v, SendReplyFlags flags);
+/**
+ * Formats the passed numeric RSValue as a string.
+ * The passed RSValue must be of type RSValue_Number.
+ */
+static size_t RSValue_NumToString(const RSValue *v, char *buf) {
+  assert(v->t == RSValue_Number);
```

Comment:
Use `RS_ASSERT` instead of the standard `assert`
```suggestion
  RS_ASSERT(v->t == RSValue_Number);
```